core/app/auth/auth.go

```diff
@@ -111,9 +111,29 @@ func ServerInterceptor(token Token) grpc.UnaryServerInterceptor {
 	}
 }
 
-// ClientInterceptor returns a grpc.UnaryClientInterceptor that adds the given
-// auth token to outgoing RPC calls.
-func ClientInterceptor(token Token) grpc.UnaryClientInterceptor {
+// StreamServerInterceptor returns a grpc.UnaryServerInterceptor that checks
```

Comment:
Typo in comment: `grpc.UnaryServerInterceptor` -> `grpc.StreamServerInterceptor`